datafusion/core/src/physical_plan/mod.rs

```diff
@@ -472,7 +508,10 @@ pub enum Distribution {
     HashPartitioned(Vec<Arc<dyn PhysicalExpr>>),
```

Comment:
Maybe we need to add partition number and schema to the `HashPartitioned ` in the future.